UnicornRender/video/include/unicorn/video/Renderer.hpp

```diff
@@ -92,15 +86,16 @@ class Renderer
     * @return true if mesh was sucessfully deleted from system and false if not
     */
     UNICORN_EXPORT virtual bool DeleteMesh(Mesh const* mesh) = 0;
+
+    //! Main view camera, must never be nullptr
+    Camera* camera;
```

Comment:
Move member declaration to the top of the list, please.

Reply:
NEVAR